src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -9482,10 +9473,8 @@ void Unit::SetPower(Powers power, int32 val)
             player->SetGroupUpdateFlag(GROUP_UPDATE_FLAG_CUR_POWER);
     }
     /*else if (Pet* pet = ToCreature()->ToPet()) TODO 6.x
-    {
         if (pet->isControlled())
-            pet->SetGroupUpdateFlag(GROUP_UPDATE_FLAG_PET_CUR_POWER);
-    }*/
+            pet->SetGroupUpdateFlag(GROUP_UPDATE_FLAG_PET_CUR_POWER);*/
```

Comment:
it looks like the whole GroupUpdateFlag stuff is unused @Shauren ?

Reply:
iirc we handle it completely wrong right now (and flags should be used in SMSG_PARTY_MEMBER_STATE_UPDATE which we dont ever send)